student.rb

```diff
@@ -3,6 +3,8 @@ class Student < Person
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
     @classroom = classroom
+    # add student to a classroom when setting classroom for a student
+    classroom.add_student(self)
```

Comment:
- [ ] It would be better if you make this a method on its own instead of adding it in initialize. It can be something like this.
```
  def add_classroom=(classroom)
    @classroom = classroom
    classroom.students.push(self) unless classroom.students.include?(self)
  end
```